src/components/pages/tours/TourEdit.tsx

```diff
@@ -70,18 +81,22 @@ const EditTour = () => {
     remove: removeItem
   }
 
+  const tourType: string = tour?.status === TourStatusType.CREATED ? 'Create' : 'Edit'
```

Comment:
Because I am not exactly sure - if `tour` is undefined here, would this expression then return `Edit`? If so, is that desired?

Reply:
yes it would return edit. I added a check if tour is defined or not. If it's undefined tourType will be undefined too.

Resolved in PR #94